packages/shared/src/s3/commands.ts

```diff
@@ -72,6 +76,9 @@ export class S3Commands extends S3Client {
   listObjects: ListObjectsV2 = (input) =>
     this.send(new ListObjectsV2Command(input));
 
+  listVersionedObjects: ListObjectVersions = (input) =>
```

Comment:
to have same name as its corresponding type (just making consistent with others)
```suggestion
  listObjectVersions: ListObjectVersions = (input) =>
```